setup.py

```diff
@@ -31,12 +31,12 @@
     download_url=DOWNLOAD_URL,
     license='All Rights Reserved.',
     packages=find_packages(),
-    # dependencies are named in requirements.pip
-    install_requires=[],
+    install_requires=['krux-stdlib', 'marathon'],
     entry_points={
         'console_scripts': [
             'marathon-api = krux_marathon_api.cli:main',
         ],
     },
     zip_safe=False,  # Don't install a single .egg file, it's harder to debug
+    python_requires='~=2.7, ~=3.6',
```

Comment:
Does this actually work on Python 3? Have we tested that?

Reply:
Yes, the test I ran above in the PR I did twice, once in 2.7.16 and once in 3.6.7.